### PR TITLE
2-tier markteplace template + shared templates

### PR DESCRIPTION
**Pull request by MSSedusch**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@MSSedusch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sedusch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: sap-2-tier-marketplace-image/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+    "nestedDeploymentName": "nestedTemplate",
+    "nestedDeploymentNameNSG": "[concat(variables('nestedDeploymentName'), 'nsg')]",
+    "nestedDeploymentNameProf": "[concat(variables('nestedDeploymentName'), 'prof')]",
+    "osDiskType": "image"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[variables('storageAccountName')]",
+      "apiVersion": "2015-06-15",
```
Comment (singhkay) [inline sap-2-tier-marketplace-image/azuredeploy.json]: apiVersion should become a variable and be reused later across the resources

File: sap-2-tier-marketplace-image/metadata.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Deploy a VM that is customized for use with SAP NetWeaver.",
+  "description": "This template allows you to deploy a VM using a operating system that is supported by SAP.",
+  "summary": "This template takes a minimum amount of parameters and deploys a VM that is customized for use with SAP NetWeaver, using the latest patched version of the selected operating system.",
+  "githubUsername": "mssedusch",
+  "dateUpdated": "2016-02-16"
```
Comment (singhkay) [inline sap-2-tier-marketplace-image/metadata.json]: Update the date on the last commit

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**